### DISCUSSION TITLE
Add central boiler control and more ...

### DISCUSSION THREAD

**Discussion by jmcollin78** [jmcollin78/versatile_thermostat#353] [Announcements]
## What's Changed 🎉 
* Add central boiler control by @jmcollin78 in https://github.com/jmcollin78/versatile_thermostat/pull/352
* Specify window action on window open detection - Issue #255 - Thanks to @Abyssmal
* Only modify _last_regulation_change if new regulation is sent to thermostats by @ikoz in https://github.com/jmcollin78/versatile_thermostat/pull/351
* Add the possibility to disable the safety mode on outdoor temperature sensor failure - Issue #343
* FIX #341 - when window state change, open_valve_percent should be resend - thanks to @nurunet

## New Contributors 👍 
* @ikoz made their first contribution in https://github.com/jmcollin78/versatile_thermostat/pull/351

**Full Changelog**: https://github.com/jmcollin78/versatile_thermostat/compare/5.2.2...5.3.3

Special thanks to [garywg](https://github.com/garywg) which helps to test the central boiler feature in real conditions and gives some extra feedback !

<hr /><em>This discussion was created from the release <a href='https://github.com/jmcollin78/versatile_thermostat/releases/tag/5.3.3'>Add central boiler control and more ...</a>.</em>

**Comment by scaytrase** [comment]
Hello! Is it possible (or planned) to attach vtherms not to single central unit but for different ones?

In my case I have signle boiler which operates on it own, since it also provides hot tap water. But for the floor heating I have several pumps (one per floor). Each pump supplies several rooms with vtherms over switches (controlling 220v valves). So in my case I need to turn on\off relevant pump if any\none of rooms is demanding heating.

**Reply by jmcollin78** [reply]
Hello @scaytrase ,

No it is not possible now and will certainly be not developed because developed in the central configuration component which is alone by construction.
